class.jetpack-gutenberg.php

```diff
@@ -9,6 +9,8 @@
 /**
  * Helper function to register a Jetpack Gutenberg block
  *
+ * @deprecated 7.1.0 Use (Gutenberg's) register_block_type() instead
+ *
```

Comment:
Lets also use `_deprecated_function( __FUNCTION__, '7.1', 'register_block_type' ); `
inside the function call so that we throw PHP Notices errors when someone used it. 
The same should be applied to the other deprecated functions

Reply:
225a8d8a7